### ISSUE TITLE
hyperscript or render problem with v1.1.0?

### ISSUE THREAD

**Issue by ChrisGitIt** [lhorie/mithril.js#1760]
Hi all,

i just upgraded to v1.1.0 but i soon got some weird errors. I just had a quick glance at the mithril internals, but it looks Iike an Zero Value (aka 0), will be discarded from the state object. A string zero value (aka "0"), seems to work ...

Here is a boiled down version:

```
<body>
    <script src="https://unpkg.com/mithril/mithril.js"></script>
    <script>
var root = document.body

function oninit(vnode) {
	vnode.state.count = '0';// WORKS -- output: 0
        //vnode.state.count = 0; // does NOT work !!! -- output: nothing (empty div)
}
function view (vnode) {
  return [
		m('div', vnode.state.count)
	]
}
var testing = {
	oninit: oninit,
	view: view
	
} 
m.render(root,m(testing));
    </script>
</body>
```

**Comment by isiahmeadows** [comment]
Dupe of #1752, fixed in #1753.

**Comment by ChrisGitIt** [comment]
Thanks a lot for your feedback! I just arrived at the same mithril code point when your gitbub informations reached me. Dang it, i wanted to commit this issue with a fix.
